docker/pipelines_runner.sh

```diff
@@ -41,6 +41,14 @@ function parse_args {
         region="$2"
         ;;
 
+      --subnetwork)
+        subnetwork="$2"
+        ;;
+
+      --privateAddress)
+        privateAddress="$2"
```

Comment:
I don't like the fact that this flag is CamleCase while all others are snake_case.
I think it's because [Dataflow](https://cloud.google.com/dataflow/docs/guides/specifying-exec-params) uses snake_case while [pipelines-tools](https://github.com/googlegenomics/pipelines-tools/blob/master/pipelines/internal/commands/run/run.go) uses CamleCase.

How about this: Let's stick to the naming of Dataflow. So the input flag that we parse will be `[no_]use_public_ips` and when we construct the `extra_args` we use `privateAddress`?

Reply:
Yes that bothered me as well, however, I matched the casing on pipeline-tools because Dataflow uses different names and casing depending on Python or Java (https://cloud.google.com/dataflow/docs/guides/specifying-networks#java).  

Since it is inconsistent in Dataflow and pipeline-tools, how about we use `public_ips true/false` for the input argument and shape it as needed in the script?

Reply:
Also using a flag and a true/false value simplifies the args parsing.

Reply:
To be honest I think adding a third naming convention only makes things more complicated. I think I prefer to stick with Dataflow Python SDK's standard, let's use `use_public_ips`.

And also true/false value to simplify the parsing is a good idea.

Reply:
Sounds good, will also make that change.